_posts/2018-04-10-building-the-peer-to-peer-internet.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: Building The Peer-to-Peer Internet
+slug: building-the-peer-to-peer-internet
+text: Workshop series at the Toronto Public Library
+location: Fort York branch of the Toronto Public Library
+locationLink: https://www.torontopubliclibrary.ca/detail.jsp?Nr=p_cat_branch_name:Fort%20York
+date: 2018-04-10
```

Comment:
why not create an event for each night? I think we want to indicate it is going on for the whole month!

Reply:
I was thinking to paste this 6 times, but may cause confusion when people click the event only to find they cannot attend because registration had to be done way ahead.

Reply:
Would strongly favour one per event, that way our event roll is an accurate reflection of the things that tomesh put on. 

While we don't have a great way to redirect for recurring events, there is one model/established pattern that we used for State of Our Networks and redirects.

Reply:
Okay I can make 6 instances of this and change the title to reflect the topic of the week, then add a note saying you must be enrolled to attend this iteration of the series.

@dcwalk what do you mean by redirects in this context? You would like a `https://tomesh.net/peer-to-peer-internet` page?

Reply:
We added a redirect to: a single page
`redirect_to: '/state-of-our-networks'` 
I could see this being helpful here because then you have 6 events, but only one place you have to update details (and so you could move all the description stuff, links to content, etc... to that one page? 

If the url is a concern: I _think_ but I could be wrong, if you add the page under `events` (currently used for pagination) it'll not muck anything up _and_ then the url will look like `tomesh.net/events/XXXXXX` rather than `tomesh.net/XXXXXX`

example: 
- https://raw.githubusercontent.com/tomeshnet/tomesh.net/master/_posts/2017-01-14-mini-conference.md
- https://raw.githubusercontent.com/tomeshnet/tomesh.net/master/state-of-our-networks.md